app/core/src/main/java/com/fsck/k9/preferences/GeneralSettingsDescriptions.java

```diff
@@ -416,6 +419,28 @@ public static int convertFromOldSize(int oldSize) {
         }
     }
 
+    /**
+     * Upgrades the settings from version 75 to 76.
+     *
+     * <p>
+     * Change default value of {@code registeredNameColor} from {@code 0xFF00008F} to {@code -638932 )} (#F6402C).
```

Comment:
The actual colors are probably not important. The reason why the colors were changed are probably more interesting to a reader in the future.

```suggestion
     * Change default value of {@code registeredNameColor} to have enough contrast in both the light and dark theme.
```